packages/loot-core/src/client/upgrade-notifications.js

```diff
@@ -55,7 +55,7 @@ export default function checkForUpgradeNotifications(
               type: 'message',
               title: 'Split transactions now support transfers & payees',
               message:
-                'The payee field is now available on split transactions, allowing you to perform transfers on individual split transactions.\n\nAll existing split transactions have a blank payee and we recommend using the tool below to set the payee from the parent. [View a video walkthrough](https://actualbudget.com/blog/split-transactions-transfer)',
+                'The payee field is now available on split transactions, allowing you to perform transfers on individual split transactions.\n\nAll existing split transactions have a blank payee and we recommend using the tool below to set the payee from the parent.',
```

Comment:
archived link: https://web.archive.org/web/20220523224938/https://actualbudget.com/blog/split-transactions-transfer
underlying YouTube video: https://www.youtube.com/watch?v=5kTtAsB0Oqk

Do you think it’s still worth linking to one or both? (IMO no since nobody will ever see this notification, just realized that after typing in the comment, but still gonna post anyway for posterity)

Reply:
I wonder if this message is even still relevant. If I'm not mistaken - it's shown when upgrading from version X to Y.

Seems to me we could just remove this code-path with the message (but I might be wrong as I've not investigated more deeply).

Reply:
Yep, none of the notifications are relevant to the OSS version but I think it’s useful to keep it here for reference if we want to add new ones.

Reply:
I have added a video link back in for reference